_i18n/en.yml

```diff
@@ -613,19 +613,19 @@ user-guides:
   import-blockchain: Importing the Monero blockchain
   monero-tools: Monero Tools
   purchasing-storing: Securely purchasing and storing Monero
-  verify-allos: Verify binaries on Linux, Mac, or Windows command line (advanced)
-  verify-windows: Verify binaries on Windows (beginner)
+  verify-allos: How to verify Monero CLI/GUI binaries using the command line (advanced)
+  verify-windows: How to verify Monero CLI/GUI binaries on Windows (beginner)
```

Comment:
If we are changing wording, maybe better use "software" instead of "binaries". The average users don't know what binaries are.